programmers-coding-test/kimjiwon/알고리즘/거스름돈.js

```diff
@@ -0,0 +1,32 @@
+function giveChange(customerMoney, goodsPrice) {
+    const unitMoney = [10000 ,5000 ,1000, 500, 100, 50, 10];
```

Comment:
돈이 여러개 들어있으므로 복수로 표현해주는 것이 좋다.
이후 반복문에서 원소를 빼내올 때 단수형으로 불러오도록 하자.
```suggestion
    const unitMoneys = [10000 ,5000 ,1000, 500, 100, 50, 10];
```